utils/docker/Dockerfile.centos.7

```
@@ -177,11 +168,15 @@ COPY src src
 USER root:root
 RUN chown -R daos_server.daos_server /home/daos
 USER daos_server:daos_server
-RUN ln -s utils/sl site_scons
+
+# select compiler to use
+ARG COMPILER=gcc
+ARG JOBS=$DEPS_JOBS
+ARG DAOS_BUILD=$DAOS_DEPS_BUILD
 
 # Build DAOS
-RUN [ "$DAOS_BUILD" != "yes" ] || \
-    (scons -j 4 install PREFIX=/opt/daos COMPILER=$COMPILER && \
+RUN ln -s utils/sl site_scons && [ "$DAOS_BUILD" != "yes" ] || \
```

Use the more natural boolean logic available in the Bourne shell, and avoid double negatives:

```suggestion
RUN ln -s utils/sl site_scons && "$DAOS_BUILD" && \
```